tests/test.py

```diff
@@ -1,5 +1,10 @@
 import time
-from check_name import CheckName
+import sys
+import os
+
+sys.path.append(
+    os.path.dirname(os.path.realpath(__file__)) + "/../file_checker")
```

Comment:
This is necessary until this project switches over to use a testing framework, such as `pytest` or `nose`.